js/admin.block-post.js

```diff
@@ -4,24 +4,23 @@
  * @package   Block_Lab
  * @copyright Copyright(c) 2019, Block Lab
  * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
- *
- * Globals wp, blockLab
  */
 
-(function( $ ) {
+/* global blockLab, jQuery */
```

Comment:
Wow, this is great. It looks like this now passes `eslint`!

What do you think of adding it to the `eslint` configuration?

```diff
diff --git a/package.json b/package.json
index c5cd510..6c7cf1a 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "dev": "cross-env BABEL_ENV=default webpack --watch",
     "build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:js": "eslint js/blocks",
+    "lint:js": "eslint js",
     "lint:js:fix": "eslint js/blocks --fix",
     "lint:php": "vendor/bin/phpcs",
     "lint:php:fix": "vendor/bin/phpcbf"
```

Also, creating an `.eslintignore` with the following will prevent it from running on the compiled files:

```js
js/editor.blocks.js
js/scripts.js
```